pkg/config/config.go

```diff
@@ -23,9 +24,11 @@ type ConfigManager struct {
 }
 
 type Config struct {
-	IgnoredVulns      []IgnoreEntry `toml:"IgnoredVulns"`
-	LoadPath          string        `toml:"LoadPath"`
-	GoVersionOverride string        `toml:"GoVersionOverride"`
+	IgnoredVulns                   []IgnoreEntry                        `toml:"IgnoredVulns"`
+	IgnoredPackageVersions         []IgnorePackageVersionEntry          `toml:"IgnoredPackageVersions"`
```

Comment:
I feel like ignore package versions and override package versions both could have the same "package matching" logic.

Can we consolidate them into the same field, and just have a bool `ignore` which if true ignores the package entirely, otherwise an override field where someone can override the license.

E.g.
```
[[Package]]
name = "pkg-name"
exactVersion = "1.0.0"
ecosystem = "Go"
ignore = false # if true ignores the package entirely
licenseOverride = ["MIT", "0BSD"]
# In the future, if someone requests we can also add a versionOverride field here, or other overrides
reason = "abc"
# major ...
# minor ...
# ...
```

Reply:
+1.

`version` (Which I think we should `exactVersion` to) is optional right? 

Additionally, would it make sense to make `license` a structured field of some sort instead? 

I.e.

```
[[Package]]
[[Package.license]]
override = ["MIT"]
```

Reply:
This makes sense to me! done.